projects/plugins/jetpack/extensions/blocks/tiled-gallery/edit.native.js

```diff
@@ -38,6 +38,7 @@ export function defaultColumnsNumber( images ) {
 const TiledGalleryEdit = props => {
 	const [ resizeObserver, sizes ] = useResizeObserver();
 	const [ maxWidth, setMaxWidth ] = useState( 0 );
+	const { width } = sizes || {};
```

Comment:
Following a conversation around whether destructuring `width` here would prevent it from changing when `sizes` changes: I set up some `console.log` calls to log both `sizes` and `width`. Those logs showed that `width` does change along with `sizes` when I change from portrait to landscape mode. 

`width` is also destructured in the same way [here](https://github.com/WordPress/gutenberg/blob/513ff939978f1c4eba9ee11c1e99c3b2a25e9a8e/packages/block-library/src/columns/edit.native.js#L107) in the columns block, where it goes onto be used in the dependency array of a `useEffect` hook [here](https://github.com/WordPress/gutenberg/blob/513ff939978f1c4eba9ee11c1e99c3b2a25e9a8e/packages/block-library/src/columns/edit.native.js#L132) (indicating it does change in value). 

Is there something I'm missing here? I just want to make sure I understand the concerns with this line before changing it.